src/ol/interaction/Extent.js

```diff
@@ -293,6 +318,9 @@ class Extent extends PointerInteraction {
    * @inheritDoc
    */
   handleDownEvent(mapBrowserEvent) {
+    if (!this.condition_(mapBrowserEvent)) {
+      return;
```

Comment:
`return false`  here to indicate that the event has not been consumed by this interaction.